iceoryx_posh/experimental/include/iox/posh/experimental/node.hpp

```diff
@@ -66,9 +66,11 @@ class NodeBuilder
 
     /// @brief Indicates whether the node shares the address space with 'RouDi', e.g. in single process applications or
     /// tests
-    IOX_BUILDER_PARAMETER(bool, shares_address_space_with_roudi, false)
+  protected:
+    bool m_shares_address_space_with_roudi{false};
 
   public:
+    NodeBuilder&& shares_address_space_with_roudi(const bool value) && noexcept;
```

Comment:
Oh, forgot about that. Could you please move the doxygen comment from line 67 and 68 to this function and add
```
/// @param[in] value to set the property
``

Reply:
Sure,.